Global/Eclipse.gitignore

```diff
@@ -49,3 +49,15 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Java class files
+**/*.class
+**/classes
```

Comment:
I'm not sure on these rules - could you tell me a bit about where they come from?